integration_tests/projects/009_execute_sql_function/fal_scripts/query_other_model.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+import io
+import os
+from functools import reduce
+
+df: pd.DataFrame = execute_sql('SELECT * FROM {{ ref("other_model")}}')
```

Comment:
This is cool! The `ref` does not get picked up by fal for Python models, though.

Reply:
That's true! Do we want it to be picked up automatically? I think it would be a great feature.